products/BellHybrid/CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ target_compile_options(BellHybrid
     PRIVATE
         $<$<COMPILE_LANGUAGE:C>:-Wno-discarded-qualifiers>
         "-Wno-error=deprecated-declarations"
-        "-Wno-prio-ctor-dtor"
```

Comment:
it's not needed now?

Reply:
This flag is now passed only to one specific CMake target(which needs it) instead of propagating it throughout the system.